x-pack/plugin/eql/qa/common/src/main/resources/test_queries_unsupported.toml

```diff
@@ -881,34 +881,6 @@ file where opcode=0 and indexOf(file_name, 'explorer.', 0) == 0'''
 expected_event_ids  = [88, 92]
 description = "check substring ranges"
 
-[[queries]]
```

Comment:
these should be moved to `test_queries_supported.toml` if they are working now

Reply:
they should be already there, test_queries_unsupported.toml file more like a blacklist for test harness